src/pages/products/sgid/boundaries/gsl-sentinel-landscape.astro

```diff
@@ -0,0 +1,75 @@
+---
+import Layout from '@layouts/DataPage.astro';
+
+import type { IMetadata, IPageMetadata } from '@models/types';
+import { ProductType, SgidCategory } from '@models/types';
+
+import HubDownloads from '@components/data/HubDownloads.astro';
+import Metadata from '@components/data/Metadata.astro';
+
+import CardWithPopularLink from '@components/page/CardWithPopularLink.astro';
+import CardWithSmallLink from '@components/page/CardWithSmallLink.astro';
+import GridForMoreResources from '@components/page/GridForMoreResources.astro';
+import GridForYouMightLike from '@components/page/GridForYouMightLike.astro';
+import Section from '@components/page/Section.astro';
+
+import { dataPages } from '@data/downloadMetadata';
+import { removeDateRange } from '@utils/data';
+
+export const metadata: IMetadata = {
+  pageTitle: 'Utah GSL Sentinel Landscape Boundary',
+  pageDescription: `The federally established boundary identifying the area of the Great Salt Lake Sentinel Landscape.`,
+  stewards: ['Department of Veterans Affairs'],
+  type: ProductType.POLYGON,
+  category: SgidCategory.BOUNDARIES,
+  secondaryCategory: SgidCategory.PLANNING,
+};
+
+const page: IPageMetadata = {
+  ...metadata,
+  updateHistory: [],
+  hub: {
+    ...dataPages[metadata.pageTitle],
+  },
```

Comment:
If you remove the slot="description" this object needs to know by adding a config: { skipInDepth: true },
```suggestion
  config: { skipInDepth: true },
  },
```